### DISCUSSION TITLE
What's the keymap of this "run /debug", I want use a key instead of click it

### DISCUSSION THREAD

**Discussion by xzbdmw** [rust-lang/rust-analyzer#16126] [Q&A] [answered]
<img width="341" alt="image" src="https://github.com/rust-lang/rust-analyzer/assets/97848247/ed336c5a-2787-44ba-88c6-3978d5024b69">

**Comment by alibektas** [comment, marked as answer]
Go to Keyboard Shortcuts ( Ctrl+K Ctrl S ) and then type `rust-analyzer`. The things of interest are `rust-analyzer.debug` and `rust-analyzer.run` if I recall correctly.
